pre_commit/pre_commit_helper.rb

```diff
@@ -7,19 +9,27 @@ def self.project_type(toplevel = `git rev-parse --show-toplevel`.strip)
     return nil
   end
 
-  def self.check_file?(file, extensions_to_ignore = [])
-    !extensions_to_ignore.include?(File.extname(file))
+  def self.check_file?(params)#file, extensions_to_ignore = [], extensions_to_include = [])
```

Comment:
Assignment Branch Condition size for check_file? is too high. [16.52/15]<br>Missing space after #.<br>Put a space before an end-of-line comment.